packages/pilot/src/pages/Transactions/Search/Search.js

```diff
@@ -171,10 +193,16 @@ class TransactionsSearch extends React.Component {
   constructor (props) {
     super(props)
 
+    const urlSearchQuery = this.props.history.location.search
```

Comment:
```suggestion
    const urlSearchQuery = props.history.location.search
```

aqui no constructo não é `this`, e sim as props que vieram como parametro